### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by aurambaj**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @aurambaj has signed the CLA. Thanks for the pull request!

File: box-content-sdk/src/main/res/values-ar/strings.xml [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <string description="" name="boxsdk_alert_dialog_cancel">إلغاء</string>
     <string description="" name="boxsdk_alert_dialog_ok">إرسال</string>
     <string description="OK on buttons" name="boxsdk_button_ok">موافق</string>
+    <string description="OK on buttons" name="boxsdk_button_okay">OKAY</string>
```
Comment (seema-at-box) [inline box-content-sdk/src/main/res/values-ar/strings.xml]: This doesn't need translation?

File: box-content-sdk/src/main/res/values-ja/strings.xml [hunk 2]
```diff
@@ -4,6 +4,7 @@
     <string description="" name="boxsdk_alert_dialog_cancel">キャンセル</string>
     <string description="" name="boxsdk_alert_dialog_ok">送信</string>
     <string description="OK on buttons" name="boxsdk_button_ok">OK</string>
+    <string description="OK on buttons" name="boxsdk_button_okay">OK</string>
```
Comment (seema-at-box) [inline box-content-sdk/src/main/res/values-ja/strings.xml]: Here too

**Comment by seema-at-box** [comment]
+1

**Pull request merged**